app/models/satisfaction.rb

```diff
@@ -0,0 +1,6 @@
+class Satisfaction < ActiveRecord::Base
+  validates :uid, presence: true, uniqueness: true
+  validates :given_at, presence: true
+  validates :delivery_partner, inclusion: { in: %w(cas cita nicab) }
```

Comment:
Wondering if delivery partners could be brought 'up a level' as this will probably be a recurring theme...